File: upeu/persona.py
class persona:
    nombre:str
    genero:str
    ocupacion:str

    def __init__(self):
        pass
    def __init__(self,nombre):
        self.nombre=obtenerNombre

    def __init__(self,nombre,genero,ocupacion):
        self.nombre=nombre
        self.genero=genero
        self.ocupacion=ocupacion

    def obtenerNombre(self):
        return self.nombre
    def obtenerGenero(self):
        return self.genero
    def obtenerOcupacion(self):
         return self.ocupacion


    def modifiacarNombre(self,nombre):
        self.nombre=nombre
    def modifiacarGenero(self,nombre):
        self.genero=genero
    def modifiacarOcupacion(self,nombre):
        self.ocupacion=ocupacion





    def obtenerNombre(self):
        return self.nombre
    def obtenerGenero(self):
        return self.genero
    def obtenerOcupacion(self):
        return self.ocupacion
    def modifiacarNombre(self, nombre):
        self.nombre=nombre
    def modifiacarGenero(self,genero):
        self.genero=genero
    def modifiacarOcupacion(self,ocupacion):
        self.ocupacion=ocupacion

myobj=persona("persona","M", "ing.civil")
print(myobj.obtenerNombre())
print(myobj.obtenerGenero())
print(myobj.obtenerOcupacion())

'''myobj=persona()
myobj.nombre="David"
print(myobj.nombre)
myobj .modifiacarNombre("Raul")
print(myobj.nombre)
print(myobj.obtenerNombre())
'''


    

File: upeu/Persona.java
package upeu;

import javafx.scene.chart.Chart;

/**
 * Persona
 */
public class Persona {
    String nombre;
    char genero;
    String ocupacion;

    Persona(){
    }

    Persona(String nombre){
        this.nombre=nombre;
    }
    Persona(String nombre,char genero,String ocupacion){
        this.nombre=nombre;
        this.genero=genero;
        this.ocupacion=ocupacion;
    }



    

    public String obtenerNombre() {
        return nombre;   
    }
    public char obtenerGenero() {
        return genero;   
    }
    public String obtenerOcupacion() {
        return ocupacion;      
    }
    public void modificarNombre(String nombre) {
        this.nombre=nombre;
    }
    public void modificarGenero(Char genero) {
        this.genero=genero;  
    }
    public void modificarOcupacion(String ocupacion) {
        this.ocupacion=ocupacion;       
    }

    
}

class principal{

    public static void main(String[] args) {
        Persona myobj=new Persona("Dario", 'M',"ing.Civi");
        System.out.println(myobj.obtenerNombre());
        System.out.println(myobj.obtenerGenero());
        System.out.println(myobj.obtenerOcupacion());
       /*  System.out.println(myobj.nombre);
        myobj.modificarNombre("Raul");
        System.out.println(myobj.nombre);
        System.out.println(myobj.obtenerNombre());*/
    }
}
